extension/httpfs/include/httpfs_curl_client.hpp

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include "duckdb/common/http_util.hpp"
+
+namespace duckdb {
+class HTTPLogger;
+class FileOpener;
+struct FileOpenerInfo;
+class HTTPState;
+
+#include <curl/curl.h>
```

Comment:
Usually we don't include header file in a namespace, since all symbols inside of the header file is extra namespace scoped.